update-tags.mjs

```diff
@@ -0,0 +1,47 @@
+import { readFile, writeFile } from 'fs/promises'
+import { exec } from 'child_process'
```

Comment:
Not directly, however it does support the following as a means to convert it to a promise:

```
const util = require('node:util');
const exec = util.promisify(require('node:child_process').exec);

async function lsExample() {
  const { stdout, stderr } = await exec('ls');
  console.log('stdout:', stdout);
  console.error('stderr:', stderr);
}
lsExample();
```

I could implement it in a similar way, but if I make use of the prettier api call this will go away